getting-started/jdbc/docker-compose-bootstrap-db.yml

```diff
@@ -22,11 +22,13 @@ services:
     # IMPORTANT: the image MUST contain the Postgres JDBC driver and EclipseLink dependencies, see README for instructions
```

Comment:
Minor: this comment seems unnecessary, as the jdbc related jars are within docker image by default now.